### ISSUE TITLE
Namespace metadata is not preserved

### ISSUE THREAD

**Issue by jsab** [gfredericks/how-to-ns#1]
Calling `lein how-to-ns check` in a project with the following namespace:

```` clojure
(ns ^:a ^:b ^:c test-record.core
  "My beautiful ns.")
````

Yields:

````
Bad ns format:
--- a/src/test_record/core.clj
+++ b/src/test_record/core.clj
@@ -1,2 +1,2 @@
-(ns ^:a ^:b ^:c test-record.core
+(ns test-record.core
   "My beautiful ns.")
````

**Comment by gfredericks** [comment]
How do you think it should be formatted?

Unfortunately this library isn't sophisticated enough to allow any flexibility in how the `ns` form is formatted (I'd be happy if it were, but it would require a more advanced impl using whatever parsing lib cljfmt uses), so to support metadata we'd have to completely specify the formatting.

**Comment by jsab** [comment]
I would say leave it as-is. Perhaps remember the text between `ns` and the namespace name and insert it as such in the resulting form.

**Comment by gfredericks** [comment]
that's exactly the sort of advanced thing that's more difficult in the current implementation.

I could imagine maybe special-casing the flag entries to use the `^:foo` syntax, and falling back to a full map (maybe pretty-printed with `clojure.pprint`?) if there's anything else.

**Comment by jsab** [comment]
Sounds perfectly reasonable.

**Comment by gfredericks** [comment]
I'll have this released on monday. Let me know if it doesn't work the way you expect.

**Comment by gfredericks** [comment]
(by which I mean you can try it out now via `lein install`)

**Comment by gfredericks** [comment]
Released with `0.1.4`.
